### ISSUE TITLE
Version consistency

### ISSUE THREAD

**Issue by keithf4** [theory/pgtap#108]
Hard to tell what the current version that should be installed is.

There's update files up to 0.97.0
The META.json & README are for 0.96.1
The pgtap.control is for 0.95.0 (this causes all new installations to be 0.95.0).

Can do a pull request to fix this, but wasn't sure if there was something else that needed to be done as part of your version management.

**Comment by theory** [comment]
> There's update files up to 0.97.0

I don't see that change. However, the next version will likely be 0.97.0.

> The META.json & README are for 0.96.1

Working version until the release of 0.97.0.

> The pgtap.control is for 0.95.0 (this causes all new installations to be 0.95.0).

I tend not to update `pgtap.control` until release. No reason not to put it at 0.96.1, though; just don't expect an upgrade script for that version, as they're designed only for releases.

**Comment by keithf4** [comment]
https://github.com/theory/pgtap/blob/master/sql/pgtap--0.96.0--0.97.0.sql

Just noticed something else. You've got an update script for 0.96.0 but not 0.96.1. You need an update script for every version you release that can be installed in the database. Even if it changes nothing in the database. Otherwise you leave a gap in the upgrade path from one version to another and the `ALTER EXTENSION ... UPDATE ...` command won't work.

I've done that a few times with pg_partman where I only changed python scripts but nothing in the sql.

**Comment by theory** [comment]
0.96.1 is a dev-only working version. It will never be released. I only bother with upgrade scripts for releases.

**Comment by keithf4** [comment]
Ok. I said that because you said 
```
No reason not to put it at 0.96.1, though
```
in reference to the control file. That means if someone installs pgtap cleanly, they will have 0.96.1 installed and no upgrade path out of it. So you should set it to 0.96.0 then.

**Comment by theory** [comment]
No one should install it from GitHub. Well, not from master, anyway.

**Comment by keithf4** [comment]
You really need to update your install instructions if that's what people are to assume. I've always installed it from the github master branch, but never went beyond the version set in the control file.

**Comment by cbandy** [comment]
Should `pgtap_version()` in the `v0.96.0` tag return `0.96`?

```
~$ git clone https://github.com/theory/pgtap.git
Cloning into 'pgtap'...

~$ cd pgtap

~/pgtap$ git checkout v0.96.0
HEAD is now at fe52f3a... Timestamp v0.96.0.

~/pgtap$ make

~/pgtap$ grep -RA2 pgtap_version sql/pgtap.sql sql/pgtap-core.sql
sql/pgtap.sql:CREATE OR REPLACE FUNCTION pgtap_version()
sql/pgtap.sql-RETURNS NUMERIC AS 'SELECT 0.95;'
sql/pgtap.sql-LANGUAGE SQL IMMUTABLE;
--
sql/pgtap-core.sql:CREATE OR REPLACE FUNCTION pgtap_version()
sql/pgtap-core.sql-RETURNS NUMERIC AS 'SELECT 0.95;'
sql/pgtap-core.sql-LANGUAGE SQL IMMUTABLE;
```

**Comment by decibel** [comment]
On 11/17/16 10:23 PM, Chris Bandy wrote:
> Should |pgtap_version()| in the |v0.96.0| tag return |0.96|?

Yeah, there were a few glitches with the 0.96.0 release. We're hoping to 
have 0.97.0 out RSN to fix those problems.
-- 
Jim Nasby, Data Architect, Blue Treble Consulting, Austin TX
Experts in Analytics, Data Architecture and PostgreSQL
Data in Trouble? Get it in Treble! http://BlueTreble.com
855-TREBLE2 (855-873-2532)

**Comment by theory** [comment]
I've released v0.98.0 and incremented to v0.99.0 in the repo. If we need to fix any issues with v0.98.0, we can branch from its tag, fix it, and release v0.98.1.
